pkg/route/ingress/ingress.go

```diff
@@ -439,11 +439,21 @@ func (c *Controller) sync(key queueKey) error {
 		if err != nil {
 			return err
 		}
+		annotations, err_annotation := json.Marshal(&route.Annotations)
+		if err_annotation != nil {
+			return err_annotation
+		}
 		data = []byte(fmt.Sprintf(`[{"op":"replace","path":"/spec","value":%s}]`, data))
 		_, err = c.routeClient.Routes(route.Namespace).Patch(context.TODO(), route.Name, types.JSONPatchType, data, metav1.PatchOptions{})
 		if err != nil {
 			errs = append(errs, err)
 		}
+
+		annotations = []byte(fmt.Sprintf(`[{"op":"replace","path":"/metadata/annotations","value":%s}]`, annotations))
+		_, err_annotation = c.routeClient.Routes(route.Namespace).Patch(context.TODO(), route.Name, types.JSONPatchType, annotations, metav1.PatchOptions{})
+		if err_annotation != nil {
+			err_annotations = append(err_annotations, err)
```

Comment:
You don't use `err_annotations` anywhere.  Why not append the error value to `errs`?